ui/app/utils/http.js

```diff
@@ -9,22 +9,29 @@ const defaultConfig = {
   redirect: 'follow',
 };
 
+export const GET = 'GET';
+export const POST = 'POST';
+export const PUT = 'PUT';
+export const DELETE = 'DELETE';
```

Comment:
Let's get rid of this from utils/http. Instead define req type constants which are specific to the request you are trying to make. Example: POLICY_FETCH, POLICY_UPDATE, ...

Reply:
Made the change.